src/web_app/js/peerconnectionclient.js

```diff
@@ -319,6 +322,18 @@ PeerConnectionClient.prototype.onIceConnectionStateChanged_ = function() {
   if (this.pc_.iceConnectionState === 'completed') {
     trace('ICE complete time: ' +
         (window.performance.now() - this.startTime_).toFixed(0) + 'ms.');
+    // Associate SSRC's with media stream tracks and user ID in the
```

Comment:
Just tested this and the remoteStreams are not set when signallingState reaches stable.